x-pack/plugins/cross_cluster_replication/server/client/elasticsearch_ccr.js

```diff
@@ -62,4 +62,28 @@ export const elasticsearchJsPlugin = (Client, config, components) => {
     needBody: true,
     method: 'DELETE'
   });
+
+  ccr.followerIndices = ca({
```

Comment:
For the bug I am currently working on, I have implemented the `/_ccr/stats` API call as part of a generic `ccr.stats` client as we might need it for several server APIs. Let's leave it here for now but we will need to remove it once I merge it. (Note for myself)